src/cat-data-service/src/axum_service/mod.rs

```diff
@@ -0,0 +1,168 @@
+//! Main entrypoint to the AXUM version of the service
```

Comment:
Shouldn't we explicitly call this `legacy_service` instead of `axum_service`?

Reply:
Ok

Reply:
Done in latest push.